src/components/timeline/TrackContextMenu.css

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+.timelineTrackContextMenuItem {
+  display: flex;
+}
+
+.timelineTrackContextMenuSpacer {
+  width: 20px;
+  flex: 1;
+}
+
+.timelineTrackContextMenuPid {
+  color: var(--grey-90-a60);
+}
+
+.react-contextmenu-item--selected .timelineTrackContextMenuPid {
+  color: white;
```

Comment:
This way of doing it wasn't particularly nice, and I just now discovered that it doesn't work properly :( Toggling the visibility by clicking will make the color jump back to gray/black, on my system at least. Perhaps there's a simpler way to do this?

Reply:
It works if I use this selector instead: `.react-contextmenu-item:hover .timelineTrackContextMenuPid`. I can update it when I address any potential review comments :-)